skimage/filters/tests/test_edges.py

```diff
@@ -497,18 +482,82 @@ def test_vertical_mask_line(grad_func):
     assert_allclose(result, expected)
 
 
-def test_range():
+# maximum Sobel 3D edge on axis 0
+MAX_SOBEL_0 = np.array([
+    [[0, 0, 0],
+     [0, 0, 0],
+     [0, 0, 0]],
+    [[0, 0, 0],
+     [0, 0, 0],
+     [0, 0, 0]],
+    [[1, 1, 1],
+     [1, 1, 1],
+     [1, 1, 1]],
+]).astype(float)
+
+# maximum Sobel 3D edge in magnitude
```

Comment:
It seems that these binary patterns are the ones that would give the maximum potential edge at center pixel?

Then, the "blobs" image has edges at all orientations, so it is expected that edge filtering it should produce at least some locations with magnitude approximately equal to this maximum.

 I would add a brief comment to this effect for future reference.

Reply:
That's exactly the logic, and you're right, a comment will help here. Will add it.